src/Parser/Warlock/Demonology/Modules/Features/SpellUsable.js

```diff
@@ -0,0 +1,45 @@
+import SPELLS from 'common/SPELLS';
+import CoreSpellUsable from 'Parser/Core/Modules/SpellUsable';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import GlobalCooldown from 'Parser/Core/Modules/GlobalCooldown';
+import T20_2set from 'Parser/Warlock/Demonology/Modules/Items/T20_2set';
+
+
+class SpellUsable extends CoreSpellUsable{
+  lastPotentialTrigger= null;
+
+  static dependencies = {
+    ...CoreSpellUsable.dependencies,
+    combatants: Combatants,
+    globalCooldown: GlobalCooldown,
```

Comment:
This dependency is unused (or there's some hidden connection I don't see and it's necessary).